src/pwncore/routes/__init__.py

```diff
@@ -1,10 +1,67 @@
 from fastapi import APIRouter
 
 from pwncore.routes import ctf, team
+from pwncore.config import config
+from pwncore.models import Problem, Team, Container, Hint, User
+import uuid
 
 # Main router (all routes go under /api)
 router = APIRouter(prefix="/api")
 
+
+@router.get("/create")
+async def init_db():
+    if config.development:
+        await Problem.create(
+            name="Invisible-Incursion",
+            description="Chod de tujhe se na ho paye",
+            author="Meetesh Saini",
+            points=300,
+            image_name="key:latest",
+            image_config={"PortBindings": {"22/tcp": [{}]}},
+        )
+        await Problem.create(
+            name="In-Plain-Sight",
+            description="A curious image with hidden secrets?",
+            author="KreativeThinker",
+            points=300,
+            image_name="key:latest",
+            image_config={"PortBindings": {"22/tcp": [{}]}},
+        )
+        await Team.create(
+            name="CID Squad" + uuid.uuid4().hex, secret_hash="veryverysecret"
+        )
+        await Team.create(
+            name="Triple A battery" + uuid.uuid4().hex, secret_hash="chotiwali"
+        )
+        await User.create(name="Aditya Jyoti", team_id=2)
```

Comment:
Given the User model is fixed, you'll have to provide the tag (reg. no.), email, and phone number for the user to be created.